app/memorize.py

```diff
@@ -8,14 +8,6 @@
 from app.memcached import memcached
 
 
-class CacheMissError(RuntimeError):
-    """
-    Exception thrown when memorized object is not found in cache and
-    error_on_miss is set to True.
-    """
-    pass
-
-
```

Comment:
It seems like the docs for the `memorize` class (line 57) need to be updated as well